summarization/extractor_output_preprocessor.py

```diff
@@ -172,13 +187,20 @@ def _get_tokenized_summary_sentence_from_index(
         return self._get_alphanumeric_tokens(word_tokenized_text)
 
     def _get_sentence_object_for_summarized_sentence(
-            self, summarized_text_index, normal_text_index):
+            self,
+            summarized_text_index,
+            normal_text_index,
+            sentence_index_in_para,
+            sentence_weight):
         ''' instantiates/initializes and returns a sentence object'''
         return SentenceWithAttributes(
             self.summarized_text[summarized_text_index],
 
-            self.normal_text_content_list[
-                normal_text_index].content_index,
+            self.normal_text_content_list[normal_text_index].content_index,
+
```

Comment:
can remove the new lines here.

Reply:
thanks for catching this.will update accordingly!